spec/support/factory.rb

```diff
@@ -4,7 +4,8 @@
   factory :user do
     sequence(:username) { |n| "jarlax#{n}" }
     sequence(:email) { |n| "jarlax#{n}@launchacademy.com" }
-    sequence(:encrypted_password) { |n| "abcdef1234#{n}" }
+    sequence(:password) { |n| "abcdef1234#{n}" }
+    sequence(:password_confirmation) { |n| "abcdef1234#{n}" }
```

Comment:
Is there a reason password needs to be unique? If not, I'd recommend just making it `password "password"`, then when filling forms for spec tests if you want the passwords to match use "password" for confirmation, if you want to purposely make it not match use "wrong password" for confirmation